docs/design-proposals/multi-tenancy/multi-tenancy-api-resource-model.md

```diff
@@ -159,14 +159,88 @@ Here are some examples:
 
 ### CRD Support
```

Comment:
Some general comments:  1) consider leaving out uncertain parts and only check in the finalized part.  2) there are some minor grammar mistakes or typos, suggest running spelling and grammar check in editor before final commit.

Reply:
Done.